### PR TITLE
[3.x] Nginx Templates

### PR DESCRIPTION
**Pull request by raymadrona**
This PR adds Nginx Templates to the SDK [https://forge.laravel.com/api-documentation#nginx-templates](https://forge.laravel.com/api-documentation#nginx-templates).
Deletion of template in not yet working which caused by unsupported Forge API `DELETE` method.

Documentation is also updated.

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Thanks for this! I have just deployed a fix for the deleting of Nginx Templates, would you mind testing that again, please?

**Comment by raymadrona** [comment]
The `DELETE` method works perfectly! Thanks.

File: src/Resources/NginxTemplate.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+     * @var int
+     */
+    public $serverId;
+
+    /**
+     * The name of the nginx template.
+     *
+     * @var string
+     */
+    public $name;
```
Comment (jbrooksuk) [inline src/Resources/NginxTemplate.php:26]: Don't we also need a property for the template's content?

Comment (raymadrona) [reply src/Resources/NginxTemplate.php:26]: Done in this commit https://github.com/laravel/forge-sdk/pull/107/commits/ca48f5d649e5972c4e264b3a4be1bb67829060a5.

**Comment by jbrooksuk** [comment]
Thanks!

**Pull request merged**